srcpkgs/wslu/template

```diff
@@ -1,11 +1,12 @@
 # Template file for 'wslu'
 pkgname=wslu
-version=3.2.4
+version=4.0.0
 revision=1
 build_style=gnu-makefile
 short_desc="Collection of utilities for Windows 10 Linux Subsystems"
 maintainer="Marcin Puc <tranzystorek.io@protonmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/wslutilities/wslu"
-distfiles="https://github.com/wslutilities/wslu/archive/v${version}.tar.gz"
-checksum=40eaadaddbab82154985966690c686679abdc59446c79555de1798fa356e1680
+distfiles="https://github.com/wslutilities/wslu/archive/refs/tags/v${version}.tar.gz"
+checksum=f3fc0191306496e8794397ba7623754b834603fd3f37bef40a858c0fe1ac3e08
+conf_files="/usr/share/wslu/conf"
```

Comment:
manpage says

> You can override the default configuration in  \'/usr/share/wslu/custom.conf\', \'$HOME/.config/wslu/conf\', or \'$HOME/.wslurc\'

so maybe this shouldn't be a conf file?

Reply:
Definitely worth discussing upstream, especially since this file looks like it belongs in /etc.

Nevertheless, typical conf files that live in /etc can also be overriden per user but that doesn't disqualify them from being conf_files, so I don't see your point

Reply:
the installed file has a big DO NOT EDIT comment at the top is my point, i guess

Reply:
You're right, I'll talk with upstream, but they only recently introduced this file, seems the "DO NOT EDIT" warning is a bit extraneous (it would be more fitting if this file was routinely autogenerated/automodified like a package lock file).

Reply:
According to https://github.com/wslutilities/wslu/issues/241#issuecomment-1229478234 this file contains essential config defaults so I'll remove it from `conf_files` in this version